GSoC-2025/spikewrap-add motion correction processing(Hargun Kaur).md

```diff
@@ -0,0 +1,96 @@
+# Spikewrap:Add motion correction processing (Hargun Kaur)
+
+## Personal details
+
+- **Full name:** Hargun Kaur  
+- **Email:** hargunkaur286@gmail.com  
+- **GitHub username:** hargunkaur286  
+- **Zulip username:** Hargun Kaur 
+- **Location & Time-zone:** Delhi, India (IST, UTC+5:30)  
+- **Personal website:** [hargunkaur.vercel.app](https://hargunkaur.vercel.app)  
+
+### Code contributions:
+- [neuroinformatics-unit/spikewrap#232](https://github.com/neuroinformatics-unit/spikewrap/pull/232)  
+- [neuroinformatics-unit/spikewrap#231](https://github.com/neuroinformatics-unit/spikewrap/pull/231)  
+- [neuroinformatics-unit/ethology#78](https://github.com/neuroinformatics-unit/ethology/pull/78)
+
+### Proposal discussion link:
+Couldn't discuss as the mentor was unavailable for pre-submission discussion.
```

Comment:
This refers to this discussion essentially, you should make this sure this PR is linked in your final submission.